corehq/apps/export/models/new.py

```diff
@@ -427,13 +423,114 @@ def generate_instance_from_schema(cls, schema, saved_export=None):
                 # Need to rebuild tags and other flags based on new build ids
                 column.update_properties_from_app_ids_and_versions(latest_app_ids_and_versions)
                 columns.append(column)
+
+            cls._insert_system_properties(schema, table, columns)
             table.columns = columns
 
             if not instance.get_table(group_schema.path):
                 instance.tables.append(table)
 
+        # Insert the parent case table
+        if isinstance(schema, CaseExportDataSchema):
+            # TODO: Would a case type without parents still have this table?
+            cls._insert_parent_case_table(instance)
+
         return instance
 
+    @classmethod
+    def _insert_system_properties(cls, schema, table, columns):
```

Comment:
we could potentially just pass the `schema.type` since that's all we're using the schema for